### PR TITLE
Ts/develop

### PR DESCRIPTION
**Pull request by leem42**
PORTALS 297 - Fixed view more behavior
PORTALS 317 - Added internal linking for consortia cards

### CODE REVIEW

File: src/lib/containers/SynapseTableCardView.tsx [hunk 1]
```diff
@@ -97,27 +102,60 @@ class SynapseTableCardView extends React.Component<SynapseTableCardViewProps, Sy
...
+                let queryRequest = this.props.getLastQueryRequest!();
+                if (!queryRequest.query) {
+                    // parent component still setting up
+                    this.getBufferData()
+                    return
+                }
+                let offset = queryRequest.query.offset!;
+                // if its a "previous" click subtract from the offset
+                // otherwise its next and we paginate forward
+                offset += 25;
```
Comment (jay-hodgson) [inline src/lib/containers/SynapseTableCardView.tsx]: pull out as constant.  have you tested with 3 pages?

Comment (leem42) [reply src/lib/containers/SynapseTableCardView.tsx]: have not, can test that now

File: src/lib/containers/SynapseTableCardView.tsx [hunk 2]
```diff
@@ -131,12 +177,23 @@ class SynapseTableCardView extends React.Component<SynapseTableCardViewProps, Sy
...
                 schema[element.name] = index;
             });
+
+        let cardLimit = 0 
+        
+        // Either the number of cards to be shown is specified by the developer in the props
+        // or this card is under the query wrapper and we handle the view more button
+        if (this.props.isQueryWrapperChild) {
+            cardLimit = this.state.cardLimit
+        } else {
```
Comment (leem42) [inline src/lib/containers/SynapseTableCardView.tsx]: clarify this

File: src/lib/containers/SynapseTableCardView.tsx [hunk 3]
```diff
@@ -131,12 +177,23 @@ class SynapseTableCardView extends React.Component<SynapseTableCardViewProps, Sy
...
+            cardLimit = limit
+        }
+        let showViewMore = this.props.isQueryWrapperChild && data.queryResult.queryResults.rows.length >= PAGE_SIZE && this.state.hasMoreData && !this.props.isLoading
```
Comment (leem42) [inline src/lib/containers/SynapseTableCardView.tsx]: refactor

**Pull request merged**